src/dashboard.meta.tsx

```diff
@@ -6,22 +6,31 @@ import styles from './dashboard.scss';
 export const createDashboardLink = db => {
   const DashboardLink: React.FC<{ basePath: string }> = ({ basePath }) => {
     return (
-      <SideNavMenu title="HIV" className={styles.noMarker} defaultExpanded={true}>
-        <SideNavMenuItem className={styles.currentNavItem} href={`${basePath}/${db.name}`}>
-          {' '}
-          HTS{' '}
-        </SideNavMenuItem>
-        <SideNavMenuItem> Care and Treatment </SideNavMenuItem>
-        <SideNavMenuItem> PMTCT </SideNavMenuItem>
-      </SideNavMenu>
+      <SideNavMenuItem className={styles.noMarker} href={`${basePath}/${db.name}`}>
+        {db.title}
+      </SideNavMenuItem>
     );
   };
   return DashboardLink;
 };
 
-export const dashboardMeta = {
+export const hts_dashboardMeta = {
   name: 'hts-summary',
   slot: 'hts-summary-dashboard-slot',
   config: { columns: 1, type: 'grid' },
-  title: 'HTS Sessions',
+  title: 'HTS ',
+};
+
+export const caretreament_dashboardMeta = {
+  name: 'care-and-treatment',
+  slot: 'care-treatment-dashboard-slot',
+  config: { columns: 1, type: 'grid' },
+  title: 'Care and Treatment',
+};
+
+export const pmtct = {
+  name: 'PMTCT',
+  slot: 'hts-summary-dashboard-slot',
```

Comment:
@larslemos won't this be a problem, you are attaching PMTCT to HTS slot... it may not but am just flagging